src/main/java/org/broadinstitute/hellbender/tools/copynumber/caller/SimpleCopyRatioCaller.java

```diff
@@ -0,0 +1,135 @@
+     * @param callingCopyRatioZScoreThreshold               z-score with respect to length-weighted standard deviation of non-log2 copy ratio
+     *                                                      in non-outlier copy-neutral segments used for calling segments
+     */
+    public SimpleCopyRatioCaller(final CopyRatioSegmentCollection copyRatioSegments,
+                                 final double neutralSegmentCopyRatioThreshold,
+                                 final double outlierNeutralSegmentCopyRatioZScoreThreshold,
+                                 final double callingCopyRatioZScoreThreshold) {
+        ParamUtils.isPositive(neutralSegmentCopyRatioThreshold, "Copy-neutral threshold must be positive.");
+        ParamUtils.isPositive(outlierNeutralSegmentCopyRatioZScoreThreshold, "Outlier z-score threshold must be positive.");
+        ParamUtils.isPositive(callingCopyRatioZScoreThreshold, "Calling z-score threshold must be positive.");
+        this.copyRatioSegments = Utils.nonNull(copyRatioSegments);
+        this.neutralSegmentCopyRatioThreshold = neutralSegmentCopyRatioThreshold;
+        this.outlierNeutralSegmentCopyRatioZScoreThreshold = outlierNeutralSegmentCopyRatioZScoreThreshold;
+        this.callingCopyRatioZScoreThreshold = callingCopyRatioZScoreThreshold;
+        callingStatistics = calculateCallingStatistics();
+    }
+
+    public CalledCopyRatioSegmentCollection makeCalls() {
+        final List<CopyRatioSegment> segments = copyRatioSegments.getRecords();
+        final List<CalledCopyRatioSegment> calledSegments = new ArrayList<>(segments.size());
+        for (final CopyRatioSegment segment : segments) {
+            final double copyRatioMean = Math.pow(2., segment.getMeanLog2CopyRatio());
+            if (Math.abs(1. - copyRatioMean) < neutralSegmentCopyRatioThreshold) {
+                calledSegments.add(new CalledCopyRatioSegment(segment, CalledCopyRatioSegment.Call.NEUTRAL));
+            } else {
+                final double copyRatioDeviation = copyRatioMean - callingStatistics.mean;
+                if (copyRatioDeviation < -callingStatistics.standardDeviation * callingCopyRatioZScoreThreshold) {
+                    calledSegments.add(new CalledCopyRatioSegment(segment, CalledCopyRatioSegment.Call.DELETION));
+                } else if (copyRatioDeviation > callingStatistics.standardDeviation * callingCopyRatioZScoreThreshold) {
+                    calledSegments.add(new CalledCopyRatioSegment(segment, CalledCopyRatioSegment.Call.AMPLIFICATION));
+                } else {
+                    calledSegments.add(new CalledCopyRatioSegment(segment, CalledCopyRatioSegment.Call.NEUTRAL));
+                }
+            }
+        }
+        return new CalledCopyRatioSegmentCollection(copyRatioSegments.getMetadata(), calledSegments);
+    }
+
+    private Statistics calculateCallingStatistics() {
+        //get the segments that fall within the copy-neutral region
```

Comment:
`//` comments need a space before the first comment text char. Also IMO it's good practice to write them as complete sentences, and capitalize/punctuate them as such.

Reply:
Thanks.  We are aiming to get this PR in ASAP so I'm not going to address style issues here.